tests/pdf_viewer/test_pdf_download.py

```diff
@@ -0,0 +1,60 @@
+import os
+import platform
+import time
+
+import pytest
+from pynput.keyboard import Controller, Key
+from selenium.webdriver import Firefox
+from selenium.webdriver.common.by import By
+
+from modules.browser_object_navigation import Navigation
+from modules.page_object import GenericPdf
+
+
+@pytest.fixture()
+def add_prefs():
+    return []
+
+
+def test_pdf_download(driver: Firefox, fillable_pdf_url: str):
```

Comment:
I added a new conftest fixture for you so you don't have to find the download directory inside the test--everyone's going to want to use that code. Check out what I did, and feel free to copy that process in the future when you need to do something that a lot of us are also going to need. Just request `downloads_folder` like you did `fillable_pdf_url` in the function signature.